src/test/java/org/ays/step_definitions/Login.java

```diff
@@ -22,8 +22,8 @@ public void open_the_institution_login_page() {
 
     @When("Enter the username and password")
     public void enter_the_username_and_password() {
-        this.pageActions.sendKeysMethod(loginPOM.getLoginUsername(), AysConfigurationProperty.InstitutionOne.AdminUserOne.EMAIL_ADDRESS);
-        this.pageActions.sendKeysMethod(loginPOM.getLoginPassword(), AysConfigurationProperty.InstitutionOne.AdminUserOne.PASSWORD);
+        this.pageActions.sendKeysMethod(loginPOM.getLoginUsername(), AysConfigurationProperty.VolunteerFoundationAdmin.EMAIL_ADDRESS);
+        this.pageActions.sendKeysMethod(loginPOM.getLoginPassword(), AysConfigurationProperty.VolunteerFoundationAdmin.PASSWORD);
```

Comment:
Nested class'ları kaldırmayalım. Yine nested class içerisinden ulaşalım. En sağlıklı yöntem bu. İlerleyen dönemde farklı durumlar için kullanılmak istenirse diye böyle ilerlememiz daha iyi olabilir. Örneğin Institution'ın ID'sine ihtiyacımız olursa onu `AysConfigurationProperty.VolunteerFoundation.ID` şeklinde çağırmamız gerekecek.
```suggestion
        this.pageActions.sendKeysMethod(loginPOM.getLoginUsername(), AysConfigurationProperty.VolunteerFoundation.Admin.EMAIL_ADDRESS);
        this.pageActions.sendKeysMethod(loginPOM.getLoginPassword(), AysConfigurationProperty.VolunteerFoundation.Admin.PASSWORD);
```